hw/vendor/pulp_platform_register_interface.vendor.hjson

```diff
@@ -7,7 +7,7 @@
 
   upstream: {
     url: "https://github.com/pulp-platform/register_interface.git",
-    rev: "e3a4955071fcf554febd4b29a7c7a52c7f1e7fc6",
+    rev: "7cf6ae7d8bc14af2c83e3178e068d79271b20815",
```

Comment:
@cousteaulecommandant for next time if you would like to revendor - just you change the commit here.
Then from X-HEEP main folder you do

`util/vendor.py hw/vendor/pulp_platform_register_interface.vendor.hjson --update`

Reply:
@danivz should know it :)